lib/cluster_conf.py

```diff
@@ -91,11 +84,9 @@ def props(self):
             BlockDeviceMappings=self.block_device_mappings
         )
 
-    #def volume(name = '/dev/sdb', size = 100, volume_type = 'gp2', delete_on_termination = True):
     def volume(self, **kwargs):
```

Comment:
Might be good to run some validations here in the future. In general boto3 provides nice errors so I didn't invest here too much, but we might verify more stuff when we are building the conf perhaps.